bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Font.java

```diff
@@ -49,11 +50,17 @@ public final class Font extends Resource {
 	 */
 	public long handle;
 
+	/**
+	 * The zoom in % of the standard resolution used for conversion of point height to pixel height
+	 * (Warning: This field is platform dependent)
+	 */
+	int zoom;
```

Comment:
I don't see any access to this field from outside the class.
```suggestion
	private int zoom;
```

Reply:
You are right. All accesses were refactored out over the last weeks.